features/webgl2-context-drawingbuffer.yml

```diff
@@ -0,0 +1,6 @@
+name: WebGL2 drawing buffer
```

Comment:
Has same support as WebGL1 `drawingBufferXXX` so we could merge these two. I feel like WebGL1/WebGL2 are very different and keeping them separate makes sense. But can be convinced otherwise.